NodeJS/Project/test/test.js

```diff
@@ -1,67 +1,72 @@
 const expect = require('chai').expect;
 const stack = require('../lib/stack.js');
 
-describe('stack#pop', function (){
-    before(function() {
-        expected = [1,2];
-    })
-    it('should return error when not using array', function() {
-        expect(() => stack.pop({})).to.throw('Data provided is not an array')
-    })
-    it('should return error when pop empty array', function() {
-        expect(() => stack.pop([])).to.throw('Cannot pop empty array')
-    })
+describe('stack', function(){
+    describe('stack#pop', function (){
+        before(function() {
+            expected = [1,2];
+        })
+        it('should return error when not using array', function() {
+            expect(() => stack.pop({})).to.throw('Data provided is not an array')
+        })
+        it('should return error when pop empty array', function() {
+            expect(() => stack.pop([])).to.throw('Cannot pop empty array')
+        })
 
-    it('should return the same array without the last element', function() {
-        current = Array(1,2,3);
-        stack.pop(current);
-        expect(current).to.have.lengthOf(expected.length);
-        for(let i=0; i<3; i++){
-            expect(current[i]).to.equal(expected[i]);
-        }       
+        it('should return the same array without the last element', function() {
+            current = Array(1,2,3);
+            stack.pop(current);
+            expect(current).to.have.lengthOf(expected.length);
+            // for(let i=0; i<3; i++){
+            //     expect(current[i]).to.equal(expected[i]);
+            // }
+            current.forEach((element, index )=> {
```

Comment:
Awesome!
Another way to write it:
` current.forEach((element, index) => expect(element).to.equal(expected[index]));`